### PR TITLE
Add support section in blueprint documentation 

### PR DESCRIPTION
**Pull request by koegel**
The section points to the forum and to the support page.

### CODE REVIEW

File: src/docs/blueprint_download.md [hunk 1]
```diff
@@ -47,3 +47,7 @@ The source code for Eclipse Theia Blueprint is available on [Github](https://git
 ## Documentation
 
 Please see [here](/docs/blueprint_documentation) for documentation on how to customize Eclipse Theia Blueprint.
+
+## Support
+
+Need help with Theia? To get support by the community go to the [Theia forum](https://community.theia-ide.org/). To get professional support for Theia see the  [support page](/support/).
```
Comment (vince-fugnitto) [inline src/docs/blueprint_download.md:53]:
```suggestion
Need help with Theia? To get support by the community go to the [Theia forum](https://community.theia-ide.org/). To get professional support for Theia see the [support page](/support/).
```

Comment (vince-fugnitto) [inline src/docs/blueprint_download.md:53]: There is also github discussions for community support, I'm not sure if we should mention it as well or if the other forum is sufficient:

- https://github.com/eclipse-theia/theia/discussions

**Review by vince-fugnitto — COMMENTED** [review]
@koegel thank you for the contribution, please be sure to squash your changes when you are ready (and remove the merge commit). Note for the future that pull-request should not really be opened from the `master` branch of a fork as it leads to issues when amending or rebasing.

**Comment by koegel** [comment]
> @koegel thank you for the contribution, please be sure to squash your changes when you are ready (and remove the merge commit). Note for the future that pull-request should not really be opened from the `master` branch of a fork as it leads to issues when amending or rebasing.

My apologies for this mess, I should have known better :)

**Comment by koegel** [comment]
Opened a new PR https://github.com/eclipse-theia/theia-website/pull/220

**Pull request closed without merging**